ray-operator/controllers/ray/utils/util.go

```diff
@@ -32,6 +31,8 @@ const (
 	DefaultDomainName   = "cluster.local"
 )
 
+var log = logf.Log.WithName("controllers").WithName("RayCluster")
```

Comment:
For the case where utility functions are called by multiple controllers, it may be preferable to use the logger propagated in the context instead, e.g.:

```go
log := ctrl.LoggerFrom(ctx)
```